libopendroneid/wifi.c

```diff
@@ -388,16 +388,15 @@ int odid_wifi_build_message_pack_nan_action_frame(ODID_UAS_Data *UAS_Data, char
 
 int odid_message_process_pack(ODID_UAS_Data *UAS_Data, uint8_t *pack, size_t buflen)
 {
-	ODID_MessagePack_encoded *msg_pack_enc;
-
-	if (sizeof(*msg_pack_enc) > buflen)
+	ODID_MessagePack_encoded *msg_pack_enc = (ODID_MessagePack_encoded *) pack;
```

Comment:
I am not fully following all the changes being done to wifi.c.
@simonwunderlich : Maybe you would be a better person to sanity check this part of the PR?